docs/main_text.md

```diff
@@ -60,31 +60,148 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
+
+### Affymetrix
+
+We have created custom gene mapping files for most of the Affymetrix platforms we support. 
+Briefly, for Brainarray supported platforms, we use the Brainarray (e.g., `hgu133plus2hsensgprobe`) and the platform-specific annotation package from Bioconductor (e.g., `hgu133plus2.db`) to generate a platform-specific mapping file that includes probe IDs, Ensembl gene IDs, gene symbols, Entrez IDs, RefSeq and Unigene identifiers. 
+The rationale for only using probes or IDs that are accounted for in the Brainarray package is two-fold: 1) Brainarray packages are updated as we learn more about the genome and 2) it allows for these submitter processed data to be more consistent with refine.bio processed data. 
+We support identifier conversion for a limited number of platforms that either do not have a Brainarray or Bioconductory annotation packages.
+
+The code for deriving these mappings and more details are available at https://github.com/AlexsLemonade/identifier-refinery.
+If you find an issue with these mappings, please [file an issue on GitHub](https://github.com/AlexsLemonade/identifier-refinery/issues) so that we can resolve it.
+If you would prefer to report issues via e-mail, you can also email [ccdl@alexslemonade.org](mailto:ccdl@alexslemonade.org).
+
+### Illumina
+
+We support conversion from Illumina BeadArray probe IDs to Ensembl gene IDs using 
+[Bioconductor Illumina Beadarray expression packages](https://www.bioconductor.org/packages/release/BiocViews.html#___IlluminaChip), 
+allowing for one-to-many mappings.
+
+TODO: Filtering for probe quality?
 
 ## Aggregations
 
+refine.bio allows users to aggregate their selected samples in two ways: by experiment or by species.
+We use the term aggregate or aggregation to refer to the process of combining _individual samples_ to form a _multi-sample_ gene expression matrix (see also: [Downloadable Files](#downloadable-files)).
+
+* **By experiment:** Samples that belong to the same experiment will become a single gene expression matrix. 
+If you have selected all samples from two experiments with 10 and 15 samples, respectively, and have chosen the `by experiment` option, you will receive two gene expression matrices with 10 and 15 samples, respectively.
+
+* **By species:** All samples assaying the same species will be aggregated into a single gene expression matrix.
+If you have selected three experiments each from human and mouse and the `by species` option, you receive two gene expression matrices that contain all human and all mouse samples, respectively.
+
+For either aggregation method, we summarize duplicate Ensembl gene IDs to the mean expression value and only include genes (rows) that are represented in all samples being aggregated.
+Note that some early generation microarrays measure fewer genes than their more recent counterparts, so their inclusion when aggregating `by species` may result in a small number of genes being returned.
```

Comment:
Do you want to note that for the compendia that we produce, we do something different? Perhaps with a TODO